lib.nix

```diff
@@ -0,0 +1,8 @@
+{ ... }@args:
+let
+  sources = import ./nix/sources.nix;
+  iohkNix = import sources.iohk-nix args;
+  # Note that this repo is using the iohk-nix nixpkgs by default
+  # A niv nixpkgs pin can override this with the following line:
+  #iohkNix = import sources.iohk-nix ({ nixpkgsOverride = sources.nixpkgs; } // args);
+in iohkNix
```

Comment:
`import ./nix/sources.nix` is already available as `(import ./nix {}).sourcePaths` so I think this file can be deleted (see suggestions below).

Reply:
Removed.